pkg/controller/podunavailablebudget/pub_pod_event_handler.go

```diff
@@ -144,12 +119,30 @@ func (p *enqueueRequestForPod) updatePod(q workqueue.RateLimitingInterface, old,
 
 }
 
+func (p *enqueueRequestForPod) reconcilePatchPubAnnotationInPod(q workqueue.RateLimitingInterface, pod *corev1.Pod) {
```

Comment:
reconcilePatchPubAnnotationInPod  -> enqueuePatchPubAnnotationRequest
this func does not do the reconcile